customer_satisfaction/README.md

```diff
@@ -0,0 +1,92 @@
+
+The Purpose of this repository is to demonstrate how [ZenML](https://github.com/zenml-io/zenml) empowers your business to build and deploy machine learning pipelines in a multitude of ways:
+
+* By offering you a framework or template to develop within
+* By integration of tools like mlflow deployment, mlflow tracking and more
+* By allowing you to easily build and deploy your machine learning pipelines 
+
+ZenML is an extensible, open-source MLOps framework to create production-ready machine learning pipelines. Built for data scientists, it has a simple, flexible syntax, is cloud- and tool-agnostic, and has interfaces/abstractions that are catered towards ML workflows.
+
+## :snake: Python Requirements
+
+Let's jump into the Python packages you need. Within the Python environment of your choice, run:
+
+```bash
+git clone https://github.com/zenml-io/zenfiles.git
+cd customer_satisfaction
+pip install -r requirements.txt
+```
+
+If you are running the `run_deployment.py` script, you will also need to install some integrations using zenml:
+
+```bash
+zenml integration install mlflow -f
+```
+
+## :thumbsup: The Task
+
+Our team at ZenML was looking for a project which is model centric, data centric and zenml centirc for our next Zenfile. During our intiial discussions we listed out several questions like What will be the usecase for this zenfile? what's the impact of this zenfile?, etc. to evaluate a zenfile. We found out that it would be really great if we build an end to end pipeline for predicting the customer satisfaction score for the next order or purchase which will help businesses to take better decisions. 
+
+As a result of this discussion we decided to build a pipeline which will predict the customer satisfaction score for the next order or purchase. We given special consideration to zenml standard pipeline and it's steps, mlflow tracking to track our metrics and parameters, mlflow deployment to deploy our model. 
+
+Our standard training pipeline consists of several steps: 
+
+* ingest_data  :- This step will ingest the data from the dataset and will create a dataframe.
+* clean_data   :- This step will clean the data and remove the unwanted columns. 
+* model_train  :- This step will train the model and will save the model using mlfow autlogging. 
+* evaluation   :- This step will evaluate the model and will save the metrics using mlfow autlogging into the artifact store.  
+
+We have another pipeline which is the deployment_pipeline.py that implements a continuous deployment workflow. It ingests and processes input data, trains a model and then (re)deploys the prediction server that serves the model if it meets some evaluation criteria.
```

Comment:
I think a lot of this would be much clearer with a well-thought-out diagram or two.